internal/manager/run.go

```diff
@@ -179,3 +206,85 @@ func runWebhookServer(mgr manager.Manager, cfg Config) {
 	}
 
 }
+
+type caManager struct {
+	client          client.Client
+	secretName      string
+	secretNamespace string
+}
+
+func (m *caManager) Start(ctx context.Context) error {
+	return m.maybeCreateCACertificate()
+}
+
+func (m *caManager) maybeCreateCACertificate() error {
+	// TODO https://github.com/Kong/gateway-operator/issues/108 this also needs to check if the CA is expired and
+	// managed, and needs to reissue it (and all issued certificates) if so
+	ca := &corev1.Secret{}
+	ctx, cancel := context.WithTimeout(context.Background(), time.Second*30)
+	defer cancel()
+	err := m.client.Get(ctx, client.ObjectKey{Namespace: m.secretNamespace, Name: m.secretName}, ca)
+	if errors.IsNotFound(err) {
+		serial, err := rand.Int(rand.Reader, big.NewInt(9999999999))
```

Comment:
You could avoid the arbitrary literal large number here and just go with the biggest possible maximum value using constants:

```suggestion
		serial, err := rand.Int(rand.Reader, big.NewInt(math.MaxInt64))
```

Reply:
Thanks, I was looking for that constant in big and concluded that it didn't exist since there was none there.